### DISCUSSION TITLE
How to fetch presigned URLs for multipart upload with object_store crate?

### DISCUSSION THREAD

**Discussion by jayatisarkar** [apache/arrow-rs#6482] [Q&A]
Currently, there is support for fetching presigned URLs to upload an entire object to s3 via this trait: [https://docs.rs/object_store/latest/object_store/signer/trait.Signer.html](https://docs.rs/object_store/latest/object_store/signer/trait.Signer.html)

What is the recommended way of doing this for multipart uploads where a presigned URL can be generated for each part similar to the AWS S3 SDK?

**Comment by tustvold** [comment]
This is not currently supported - https://github.com/apache/arrow-rs/issues/5694
